psi4/src/psi4/libmints/integral.h

```diff
@@ -499,6 +499,9 @@ class PSI_API IntegralFactory {
     /// Returns an erf complement ERI integral object (omega integral)
     virtual TwoBodyAOInt* erf_complement_eri(double omega, int deriv = 0, bool use_shell_pairs = true, bool needs_exchange = false);
 
+    /// Returns an Yukawa integral object (Slater-type geminal times Coulomb, e^(-z*r)/r)
+    virtual TwoBodyAOInt* yukawa_eri(double zeta, int deriv = 0, bool use_shell_pairs = true, bool needs_exchange = false);
```

Comment:
Returning a raw pointer is dangerous. Can we possibly make this a `unique_ptr`, or will that create issues elsewhere down the line?

Reply:
Leaving it as a raw pointer follows the pattern of the other integrals. Another PR could be done that makes the conversion to a unique_ptr.